src/app/onion/learning-object-builder/builder-store.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@angular/core';
 import { LearningObject, LearningOutcome, User } from '@cyber4all/clark-entity';
 import { AuthService } from 'app/core/auth.service';
 import { Subject, BehaviorSubject } from 'rxjs';
-import { debounceTime, takeUntil } from 'rxjs/operators';
+import { debounceTime, takeUntil, isEmpty } from 'rxjs/operators';
```

Comment:
Are we using this isEmpty anywhere? If not we should remove it to prevent additional bloat.